src/ucp/wireup/wireup.c

```diff
@@ -1031,9 +1036,27 @@ ucp_wireup_check_config_intersect(ucp_ep_h ep,
                     (lane == ucp_ep_get_cm_lane(ep))));
     }
 
+    if (!(ep->flags & UCP_EP_FLAG_INTERNAL)) {
+        cm_wireup_ep = ucp_ep_get_cm_wireup_ep(ep);
+        ucs_assert(cm_wireup_ep != NULL);
+
+        memcpy(old_dst_rsc_indices, cm_wireup_ep->dst_rsc_indices,
+               sizeof(old_dst_rsc_indices));
+        for (lane = 0; lane < new_key->num_lanes; ++lane) {
```

Comment:
can we move it outside the if?

Reply:
unfortunately, no
since we no need to do it for TMP EP